### PR TITLE
Minor edits applied

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
+ Minor edits in files
+ Fix minor issues in readme and docstring.

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@ vulture>=1.0
 bandit>=1.5.1
 pydocstyle>=3.0.0
 pytest>=4.3.1
-pytest-cov>=2.6.1
\ No newline at end of file
+pytest-cov>=2.6.1
```
Comment (AHReccese) [inline dev-requirements.txt:9]: what is the difference?
@sadrasabouri

File: README.md [hunk 2]
```diff
@@ -123,4 +113,4 @@ Give a ⭐️ if this project helped you!
 ### Donate to our project
 If you do like our project and we hope that you do, can you please support us? Our project is not and is never going to be working for profit. We need the money just so we can continue doing what we do ;-) .			
 
-<a href="https://openscilab.com/#donation" target="_blank"><img src="https://github.com/openscilab/reserver/raw/main/otherfiles/donation.png" height="90px" width="270px" alt="Reserver Donation"></a>
\ No newline at end of file
+<a href="https://openscilab.com/#donation" target="_blank"><img src="https://github.com/openscilab/reserver/raw/main/otherfiles/donation.png" height="90px" width="270px" alt="Reserver Donation"></a>
```
Comment (AHReccese) [inline README.md:116]: what is the difference?

File: tests/test_reserver.py [hunk 3]
```diff
@@ -5,22 +5,22 @@
...
     assert uploader.upload(get_random_name()) == False
 
 def test_valid_package_valid_credentials():
     # test not reserved name -> correct credentials
-    # uploader = PyPIUploader(test_pypi_token, test_pypi= True)
+    # uploader = PyPIUploader(test_pypi_token, test_pypi=True)
     # uploader.upload_to_pypi(get_random_name())
-    assert True == True
\ No newline at end of file
+    assert True == True
```
Comment (AHReccese) [inline tests/test_reserver.py:26]: same?

File: dev-requirements.txt (hunk 1 above)
Comment (sadrasabouri) [reply dev-requirements.txt:9]: Trailing white space and/or new line.

File: README.md (hunk 2 above)
Comment (sadrasabouri) [reply README.md:116]: There is a weird bug that adds a tiny blue line below buttons if we use the former format.

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
 requests>=1.0.0
 setuptools>=40.8.0
 wheel>=0.40.0
-twine>=4.0.0
\ No newline at end of file
+twine>=4.0.0
```
[the remark this answers is not in the document]
Comment (sadrasabouri) [reply requirements.txt:4]: Trailing white space and/or new line.

File: tests/test_reserver.py (hunk 3 above)
Comment (sadrasabouri) [reply tests/test_reserver.py:26]: Trailing white space and/or new line.

**Review by AHReccese — APPROVED** [review]

**Pull request merged**